qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VerifyLoginSteps.java

```diff
@@ -39,43 +44,43 @@ public void verifyModulesAvailableOnHomePage() {
         assertTrue(homePage.isSystemAdministrationAppPresent());
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on homepage after login as clerk")
     public void verifyClerkModulesAvailableOnHomePage() {
         goToLoginPage().loginAsClerk();
-        initiateHomePage();
         assertTrue(homePage.isActiveVisitsAppPresent());
         assertTrue(homePage.isAppointmentSchedulingAppPresent());
         assertTrue(homePage.isRegisterPatientCustomizedForRefAppPresent());
+        assertFalse(homePage.isConfigureMetadataAppPresent());
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on home page after login as doctor")
     public void verifyDoctorModulesAvailableOnHomePage() {
         goToLoginPage().loginAsDoctor();
-        initiateHomePage();
         assertTrue(homePage.isFindAPatientAppPresent());
         assertTrue(homePage.isActiveVisitsAppPresent());
         assertTrue(homePage.isAppointmentSchedulingAppPresent());
```

Comment:
Is the doctor allowed to see the systems administrator app?

Reply:
All apps presented here are from homePage which the doctor is allowed to see.

Reply:
Does the doctor have the system developer privileges?

Reply:
No he doesnot.

Reply:
Then why haven't you taken care of it in your assertions?

Reply:
@dkayiwa  you can only view only these three apps after logging in as Doctor.

Reply:
Is there anything in your test which proves that you cannot view the others?

Reply:
No thanks.  Updated all the required scenarios.